tests/test_user.py

```diff
@@ -811,14 +813,14 @@ def test_deep_tree_check_role(self):
         staff_top = db.session.merge(staff_top)
         for perm in ('view', 'edit'):
             for patient in (
-                patient_w_id, patient_x_id, patient_y_id, patient_z_id):
+                    patient_w_id, patient_x_id, patient_y_id, patient_z_id):
```

Comment:
Can you put `):` on its own line, at the same indentation as `for` to make it more obvious what's in the block and what's part of the loop definition